### ISSUE TITLE
Failed to load css and js files under "web" folder

### ISSUE THREAD

**Issue by darkgeek** [wallabag/wallabag#1613]
I use `composer` to install `wallabag` *2.0.0-alpha.1* version on `FreeBSD` *10.2* amd64 computer. Installation is flawless, however, when I run the server and open the login page for the first time,  it fails to load any css or js files, complaining `404 not found` error.

These are what I do with `wallabag`:
```bash
% composer create-project wallabag/wallabag wallabag 2.0.0-alpha.1
...Skip the output of dependency installation since no errors are reported...

% cd wallabag && php app/console wallabag:install
Installing Wallabag...

Step 1 of 4. Checking system requirements.
+---------+--------+----------------+
| Checked | Status | Recommendation |
+---------+--------+----------------+
| PCRE    | OK!    |                |
| DOM     | OK!    |                |
+---------+--------+----------------+
Success! Your system can run Wallabag properly.

Step 2 of 4. Setting up database.
Creating database and schema, clearing the cache

Step 3 of 4. Administration setup.
Would you like to create a new user ? (y/N)y
Username (default: wallabag) :
Password (default: wallabag) :
Email:

Step 4 of 4. Installing assets.

Wallabag has been successfully installed.
Just execute `php app/console server:run` for using wallabag: http://localhost:8000

% ls ./web
app.php     app_dev.php bundles     robots.txt  themes      uploads

% php app/console server:run
Server running on http://127.0.0.1:8000

Quit the server with CONTROL-C.
```
Then I open this address with `Firefox` *43.0.4*:
![2016-01-22-013018_1278x759_scrot_meitu_2](https://cloud.githubusercontent.com/assets/1317617/12491113/22d336ae-c0b4-11e5-8d6d-428601ed3a8a.jpg)

And the php modules installed on the computer are listed as follow:
```
% pkg info php\*
php-composer-1.0.0.a11_1
php56-5.6.17
php56-bz2-5.6.17
php56-ctype-5.6.17
php56-curl-5.6.17
php56-dom-5.6.17
php56-exif-5.6.17
php56-fileinfo-5.6.17
php56-filter-5.6.17
php56-gd-5.6.17
php56-gettext-5.6.17
php56-hash-5.6.17
php56-iconv-5.6.17
php56-json-5.6.17
php56-ldap-5.6.17
php56-mbstring-5.6.17_1
php56-mysql-5.6.17
php56-openssl-5.6.17
php56-pdo-5.6.17
php56-pdo_mysql-5.6.17
php56-pdo_sqlite-5.6.17
php56-phar-5.6.17
php56-posix-5.6.17
php56-session-5.6.17
php56-simplexml-5.6.17
php56-sqlite3-5.6.17
php56-tidy-5.6.17
php56-tokenizer-5.6.17
php56-wddx-5.6.17
php56-xml-5.6.17
php56-xmlreader-5.6.17
php56-xmlwriter-5.6.17
php56-xsl-5.6.17
php56-zip-5.6.17
php56-zlib-5.6.17
```
Could you help figure it out, please?

**Comment by Asara** [comment]
Just to give some more context, it seems as if the themes are never copied into the web/themes folder.  While web/themes exists, there is nothing under that dir.

**Comment by j0k3r** [comment]
@darkgeek Could please paste lines that are in "...Skip the output of dependency installation since no errors are reported..." ? In these lines there is the symlink process which will copy assets to the right folder.

Could you check what's inside `web/themes/` ?

@Asara It should have symlink to themes inside this folder as in the repository https://github.com/wallabag/wallabag/tree/v2/web/themes

```
j0k@mbp:~/Sites/github/wallabag$ tree web/themes/
web/themes/
├── _global -> ../../src/Wallabag/CoreBundle/Resources/views/themes/_global/public/
├── baggy -> ../../src/Wallabag/CoreBundle/Resources/views/themes/baggy/public/
└── material -> ../../src/Wallabag/CoreBundle/Resources/views/themes/material/public/

3 directories, 0 files
```

**Comment by darkgeek** [comment]
@j0k3r OK, I reinstall it via composer, and the following are the outputs:
```bash
% composer create-project wallabag/wallabag wallabag 2.0.0-alpha.1
Installing wallabag/wallabag (2.0.0-alpha.1)
  - Installing wallabag/wallabag (2.0.0-alpha.1)
    Downloading: 100%         

Created project in wallabag
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
  - Installing behat/transliterator (v1.1.0)
    Loading from cache

  - Installing doctrine/collections (v1.3.0)
    Loading from cache

  - Installing doctrine/lexer (v1.0.1)
    Loading from cache

  - Installing doctrine/inflector (v1.1.0)
    Loading from cache

  - Installing doctrine/cache (v1.6.0)
    Loading from cache

  - Installing doctrine/annotations (v1.2.7)
    Loading from cache

  - Installing doctrine/common (v2.6.1)
    Loading from cache

  - Installing doctrine/dbal (v2.5.3)
    Loading from cache

  - Installing twig/twig (v1.23.1)
    Loading from cache

  - Installing psr/log (1.0.0)
    Loading from cache

  - Installing symfony/symfony (v2.7.8)
    Loading from cache

  - Installing doctrine/doctrine-cache-bundle (1.2.2)
    Loading from cache

  - Installing ezyang/htmlpurifier (v4.7.0)
    Loading from cache

  - Installing friendsofsymfony/oauth2-php (1.1.1)
    Loading from cache

  - Installing friendsofsymfony/oauth-server-bundle (1.4.2)
    Loading from cache

  - Installing willdurand/negotiation (1.5.0)
    Loading from cache

  - Installing willdurand/jsonp-callback-validator (v1.1.0)
    Loading from cache

  - Installing friendsofsymfony/rest-bundle (1.7.7)
    Loading from cache

  - Installing friendsofsymfony/user-bundle (dev-master e5e7a2b)
    Cloning e5e7a2b8984da8dfedaf44adc7e5f60a62ad280c

  - Installing grandt/relativepath (1.0.2)
    Loading from cache

  - Installing grandt/binstring (1.0.0)
    Loading from cache

  - Installing grandt/phpzipmerge (1.0.4)
    Loading from cache

  - Installing phpzip/phpzip (2.0.8)
    Loading from cache

  - Installing grandt/phpresizegif (1.0.3)
    Loading from cache

  - Installing grandt/phpepub (4.0.3)
    Loading from cache

  - Installing react/promise (v2.2.1)
    Loading from cache

  - Installing guzzlehttp/streams (3.0.0)
    Loading from cache

  - Installing guzzlehttp/ringphp (1.1.0)
    Loading from cache

  - Installing hoa/core (2.15.11.09)
    Loading from cache

  - Installing hoa/iterator (1.15.10.29)
    Loading from cache

  - Installing hoa/visitor (1.15.08.17)
    Loading from cache

  - Installing hoa/ustring (3.15.11.09)
    Loading from cache

  - Installing hoa/regex (0.15.08.13)
    Loading from cache

  - Installing hoa/math (0.15.10.26)
    Loading from cache

  - Installing hoa/stream (0.15.10.26)
    Loading from cache

  - Installing hoa/file (0.15.11.09)
    Loading from cache

  - Installing hoa/compiler (2.15.10.29)
    Loading from cache

  - Installing hoa/ruler (1.15.11.09)
    Loading from cache

  - Installing incenteev/composer-parameter-handler (v2.1.2)
    Loading from cache

  - Installing tecnickcom/tcpdf (6.2.12)
    Loading from cache

  - Installing smalot/pdfparser (v0.9.25)
    Loading from cache

  - Installing simplepie/simplepie (1.3.1)
    Loading from cache

  - Installing neitanod/forceutf8 (v2.0)
    Loading from cache

  - Installing monolog/monolog (1.17.2)
    Loading from cache

  - Installing j0k3r/safecurl (2.0.2)
    Loading from cache

  - Installing j0k3r/php-readability (v1.0.9)
    Loading from cache

  - Installing j0k3r/graby-site-config (1.0.11)
    Loading from cache

  - Installing htmlawed/htmlawed (1.1.19)
    Loading from cache

  - Installing guzzlehttp/guzzle (5.3.0)
    Loading from cache

  - Installing j0k3r/graby (1.0.8)
    Loading from cache

  - Installing jdorn/sql-formatter (v1.2.17)
    Loading from cache

  - Installing phpoption/phpoption (1.5.0)
    Loading from cache

  - Installing phpcollection/phpcollection (0.4.0)
    Loading from cache

  - Installing jms/parser-lib (1.0.0)
    Loading from cache

  - Installing jms/metadata (1.5.1)
    Loading from cache

  - Installing jms/serializer (0.16.0)
    Loading from cache

  - Installing kphoen/rulerz (0.16.2)
    Loading from cache

  - Installing kphoen/rulerz-bundle (0.11.0)
    Loading from cache

  - Installing doctrine/instantiator (1.0.5)
    Loading from cache

  - Installing doctrine/orm (v2.5.3)
    Loading from cache

  - Installing lexik/form-filter-bundle (v4.0.2)
    Loading from cache

  - Installing liip/theme-bundle (1.1.3)
    Loading from cache

  - Installing mgargano/simplehtmldom (1.5)
    Loading from cache

  - Installing michelf/php-markdown (1.6.0)
    Loading from cache

  - Installing nelmio/api-doc-bundle (2.11.1)
    Loading from cache

  - Installing nelmio/cors-bundle (1.4.1)
    Loading from cache

  - Installing pagerfanta/pagerfanta (v1.0.3)
    Loading from cache

  - Installing sonata-project/google-authenticator (1.0.2)
    Loading from cache

  - Installing scheb/two-factor-bundle (v1.4.7)
    Loading from cache

  - Installing sensiolabs/security-checker (v3.0.2)
    Loading from cache

  - Installing sensio/distribution-bundle (v3.0.34)
    Loading from cache

  - Installing sensio/framework-extra-bundle (v3.0.12)
    Loading from cache

  - Installing gedmo/doctrine-extensions (v2.4.10)
    Loading from cache

  - Installing stof/doctrine-extensions-bundle (v1.2.1)
    Loading from cache

  - Installing kriswallsmith/assetic (v1.3.2)
    Loading from cache

  - Installing symfony/assetic-bundle (v2.7.1)
    Loading from cache

  - Installing symfony/monolog-bundle (v2.8.2)
    Loading from cache

  - Installing swiftmailer/swiftmailer (v5.4.1)
    Loading from cache

  - Installing symfony/swiftmailer-bundle (v2.3.9)
    Loading from cache

  - Installing twig/extensions (v1.3.0)
    Loading from cache

  - Installing wallabag/php-mobi (1.0.1)
    Loading from cache

  - Installing willdurand/hateoas (v2.6.0)
    Loading from cache

  - Installing jms/serializer-bundle (0.13.0)
    Loading from cache

  - Installing willdurand/hateoas-bundle (0.5.0)
    Loading from cache

  - Installing doctrine/doctrine-bundle (v1.5.2)
    Loading from cache

  - Installing doctrine/data-fixtures (v1.1.1)
    Loading from cache

  - Installing doctrine/doctrine-fixtures-bundle (v2.2.1)
    Loading from cache

  - Installing phpdocumentor/reflection-docblock (2.0.4)
    Loading from cache

  - Installing phpunit/php-token-stream (1.4.8)
    Loading from cache

  - Installing sebastian/version (1.0.6)
    Loading from cache

  - Installing sebastian/global-state (1.1.1)
    Loading from cache

  - Installing sebastian/recursion-context (1.0.2)
    Loading from cache

  - Installing sebastian/exporter (1.2.1)
    Loading from cache

  - Installing sebastian/environment (1.3.3)
    Loading from cache

  - Installing sebastian/diff (1.4.1)
    Loading from cache

  - Installing sebastian/comparator (1.2.0)
    Loading from cache

  - Installing phpunit/php-text-template (1.2.1)
    Loading from cache

  - Installing phpunit/phpunit-mock-objects (2.3.8)
    Loading from cache

  - Installing phpunit/php-timer (1.0.7)
    Loading from cache

  - Installing phpunit/php-file-iterator (1.4.1)
    Loading from cache

  - Installing phpunit/php-code-coverage (2.2.4)
    Loading from cache

  - Installing phpspec/prophecy (v1.5.0)
    Loading from cache

  - Installing phpunit/phpunit (4.8.21)
    Loading from cache

  - Installing sensio/generator-bundle (v2.5.3)
    Loading from cache

  - Installing symfony/phpunit-bridge (v2.7.8)
    Loading from cache

friendsofsymfony/oauth-server-bundle suggests installing doctrine/mongodb-odm-bundle (*)
friendsofsymfony/oauth-server-bundle suggests installing propel/propel-bundle (If you want to use Propel with Symfony2, then you will have to install the PropelBundle)
friendsofsymfony/oauth-server-bundle suggests installing willdurand/propel-typehintable-behavior (The Typehintable behavior is useful to add type hints on generated methods, to be compliant with interfaces)
friendsofsymfony/user-bundle suggests installing willdurand/propel-typehintable-behavior (Needed when using the propel implementation)
hoa/core suggests installing hoa/cli (To use the `hoa` script.)
monolog/monolog suggests installing aws/aws-sdk-php (Allow sending log messages to AWS services like DynamoDB)
monolog/monolog suggests installing doctrine/couchdb (Allow sending log messages to a CouchDB server)
monolog/monolog suggests installing ext-amqp (Allow sending log messages to an AMQP server (1.0+ required))
monolog/monolog suggests installing ext-mongo (Allow sending log messages to a MongoDB server)
monolog/monolog suggests installing graylog2/gelf-php (Allow sending log messages to a GrayLog2 server)
monolog/monolog suggests installing php-console/php-console (Allow sending log messages to Google Chrome)
monolog/monolog suggests installing raven/raven (Allow sending log messages to a Sentry server)
monolog/monolog suggests installing rollbar/rollbar (Allow sending log messages to Rollbar)
monolog/monolog suggests installing ruflin/elastica (Allow sending log messages to an Elastic Search server)
monolog/monolog suggests installing videlalvaro/php-amqplib (Allow sending log messages to an AMQP server using php-amqplib)
kphoen/rulerz suggests installing elasticsearch/elasticsearch (To execute rules as Elasticsearch queries)
kphoen/rulerz suggests installing kphoen/rulerz-spec-builder (If you want a specification builder)
kphoen/rulerz suggests installing pomm-project/model-manager (To execute rules as Pomm queries)
kphoen/rulerz suggests installing solarium/solarium (To execute rules as Solr queries)
nelmio/api-doc-bundle suggests installing dunglas/api-bundle (For making use of resources definitions of DunglasApiBundle.)
pagerfanta/pagerfanta suggests installing doctrine/mongodb-odm (To use the DoctrineODMMongoDBAdapter.)
pagerfanta/pagerfanta suggests installing doctrine/phpcr-odm (To use the DoctrineODMPhpcrAdapter. >= 1.1.0)
pagerfanta/pagerfanta suggests installing mandango/mandango (To use the MandangoAdapter.)
pagerfanta/pagerfanta suggests installing propel/propel1 (To use the PropelAdapter)
pagerfanta/pagerfanta suggests installing solarium/solarium (To use the SolariumAdapter.)
sensio/framework-extra-bundle suggests installing symfony/psr-http-message-bridge (To use the PSR-7 converters)
gedmo/doctrine-extensions suggests installing doctrine/mongodb-odm (to use the extensions with the MongoDB ODM)
stof/doctrine-extensions-bundle suggests installing doctrine/mongodb-odm-bundle (to use the MongoDB ODM extensions)
kriswallsmith/assetic suggests installing leafo/lessphp (Assetic provides the integration with the lessphp LESS compiler)
kriswallsmith/assetic suggests installing leafo/scssphp (Assetic provides the integration with the scssphp SCSS compiler)
kriswallsmith/assetic suggests installing leafo/scssphp-compass (Assetic provides the integration with the SCSS compass plugin)
kriswallsmith/assetic suggests installing patchwork/jsqueeze (Assetic provides the integration with the JSqueeze JavaScript compressor)
kriswallsmith/assetic suggests installing ptachoire/cssembed (Assetic provides the integration with phpcssembed to embed data uris)
symfony/assetic-bundle suggests installing kriswallsmith/spork (to be able to dump assets in parallel)
jms/serializer-bundle suggests installing jms/di-extra-bundle (Required to get lazy loading (de)serialization visitors, ~1.3)
doctrine/data-fixtures suggests installing doctrine/mongodb-odm (For loading MongoDB ODM fixtures)
doctrine/data-fixtures suggests installing doctrine/phpcr-odm (For loading PHPCR ODM fixtures)
phpdocumentor/reflection-docblock suggests installing dflydev/markdown (~1.0)
phpdocumentor/reflection-docblock suggests installing erusev/parsedown (~1.0)
sebastian/global-state suggests installing ext-uopz (*)
phpunit/phpunit-mock-objects suggests installing ext-soap (*)
phpunit/php-code-coverage suggests installing ext-xdebug (>=2.2.1)
phpunit/phpunit suggests installing phpunit/php-invoker (~1.1)
Generating autoload files
> Incenteev\ParameterHandler\ScriptHandler::buildParameters
Creating the "app/config/parameters.yml" file
Some parameters are missing. Please provide them.
database_driver (pdo_sqlite): 
database_host (127.0.0.1): 
database_port (null): 
database_name (symfony): 
database_user (root): 
database_password (null): 102030
database_path ('%kernel.root_dir%/../data/db/wallabag.sqlite'): 
database_table_prefix (wallabag_): 
test_database_driver (pdo_sqlite): 
test_database_host (127.0.0.1): 
test_database_port (null): 
test_database_name (null): wallabag_test
test_database_user (null): root
test_database_password (null): 102030
test_database_path ('%kernel.root_dir%/../data/db/wallabag_test.sqlite'): 
mailer_transport (smtp): 
mailer_host (127.0.0.1): 
mailer_user (null): 
mailer_password (null): 
locale (en): 
secret (ThisTokenIsNotSoSecretChangeIt): 102030
app.version (2.0.0-alpha): 
twofactor_auth (true): 
twofactor_sender (no-reply@wallabag.org): 
warning_message ("You're trying wallabag v2, which is in alpha version. If you find a bug, please have a look to <a href=\"https://github.com/wallabag/wallabag/issues\">our issues list</a> and <a href=\"https://github.com/wallabag/wallabag/issues/new\">open a new if necessary</a>\n"): 
download_pictures (false): 
share_twitter (true): 
share_mail (true): 
share_shaarli (true): 
shaarli_url ('http://myshaarli.com'): 
share_diaspora (true): 
diaspora_url ('http://diasporapod.com'): 
flattr (true): 
carrot (true): 
show_printlink (true): 
export_epub (true): 
export_mobi (true): 
export_pdf (true): 
wallabag_url ('http://v2.wallabag.org'): 
wallabag_support_url ('https://www.wallabag.org/pages/support.html'): 
items_on_page (12): 
theme (material): 
language (en): 
from_email (no-reply@wallabag.org): 
rss_limit (50): 
pocket_consumer_key (xxxxxxxx): 
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::buildBootstrap
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::clearCache
Clearing the cache for the dev environment with debug true
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::installAssets
Trying to install assets as symbolic links.
Installing assets for Symfony\Bundle\FrameworkBundle into web/bundles/framework
The assets were installed using symbolic links.
Installing assets for Nelmio\ApiDocBundle into web/bundles/nelmioapidoc
The assets were installed using symbolic links.
Installing assets for Sensio\Bundle\DistributionBundle into web/bundles/sensiodistribution
The assets were installed using symbolic links.
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::installRequirementsFile

% cd wallabag && find web && ls -l web
web
web/.htaccess
web/app.php
web/app_dev.php
web/robots.txt
web/themes
web/uploads
web/uploads/import
web/uploads/import/.gitkeep
web/bundles
web/bundles/framework
web/bundles/nelmioapidoc
web/bundles/sensiodistribution
total 24
-rw-r--r--  1 justin  justin  1014 Jan  8 05:15 app.php
-rw-r--r--  1 justin  justin  1182 Jan  8 05:15 app_dev.php
drwxr-xr-x  2 justin  justin   512 Jan 22 17:00 bundles
-rw-r--r--  1 justin  justin    25 Jan  8 05:15 robots.txt
drwxr-xr-x  2 justin  justin   512 Jan 22 16:55 themes
drwxr-xr-x  3 justin  justin   512 Jan 22 16:55 uploads
```

**Comment by j0k3r** [comment]
@darkgeek could you also `ls -l web/themes` ?

**Comment by darkgeek** [comment]
@j0k3r 
```bash
% ls -l web/themes/
total 0
```

**Comment by j0k3r** [comment]
Gosh, so it seems you have the same problem as @Asara :disappointed: 
Could you re-install wallabag using:

```
composer create-project wallabag/wallabag wallabag 2.x-dev
```

And before every thing else, `ls -l web/themes/`?

**Comment by darkgeek** [comment]
@j0k3r I reinstall it but seems that some errors are reported :worried::
```bash
% composer create-project wallabag/wallabag wallabag 2.x-dev
Installing wallabag/wallabag (2.x-dev 6c32aaae95caf6fcd6740fa1a78f8af31bb6370f)
  - Installing wallabag/wallabag (2.x-dev 6c32aaa)
    Cloning 6c32aaae95caf6fcd6740fa1a78f8af31bb6370f
    Failed to download wallabag/wallabag from source: The process "git clone --no-checkout 'git://github.com/wallabag/wallabag.git' 'wallabag/' && cd 'wallabag/' && git remote add composer 'git://github.com/wallabag/wallabag.git' && git fetch composer" exceeded the timeout of 300 seconds.
    Now trying to download from dist
  - Installing wallabag/wallabag (2.x-dev 6c32aaa)
    Loading from cache

Created project in wallabag
Loading composer repositories with package information
Installing dependencies (including require-dev)
PHP Fatal error:  Allowed memory size of 1073741824 bytes exhausted (tried to allocate 32 bytes) in phar:///usr/local/bin/composer.phar/src/Composer/DependencyResolver/Decisions.php on line 66

Fatal error: Allowed memory size of 1073741824 bytes exhausted (tried to allocate 32 bytes) in phar:///usr/local/bin/composer.phar/src/Composer/DependencyResolver/Decisions.php on line 66

% cd wallabag/ && ls -l web/themes/
total 0
```
My computer has only 1GB memory, so maybe this is the cause?

**Comment by j0k3r** [comment]
> My computer has only 1GB memory, so maybe this is the cause?

Yeah it should be the problem.
I'll try something about symlink in theme folder.

**Comment by bdunogier** [comment]
@darkgeek could you try `composer create-project --no-dev wallabag/wallabag wallabag 2.x-dev` to see if it makes a difference with the memory usage error ?

**Comment by darkgeek** [comment]
@bdunogier Well, the errors remain:
```bash
% composer create-project --no-dev wallabag/wallabag wallabag 2.x-dev
Installing wallabag/wallabag (2.x-dev 6c32aaae95caf6fcd6740fa1a78f8af31bb6370f)
  - Installing wallabag/wallabag (2.x-dev 6c32aaa)
    Cloning 6c32aaae95caf6fcd6740fa1a78f8af31bb6370f
    Failed to download wallabag/wallabag from source: The process "git clone --no-checkout 'git://github.com/wallabag/wallabag.git' 'wallabag/' && cd 'wallabag/' && git remote add composer 'git://github.com/wallabag/wallabag.git' && git fetch composer" exceeded the timeout of 300 seconds.
    Now trying to download from dist
  - Installing wallabag/wallabag (2.x-dev 6c32aaa)
    Loading from cache

Created project in wallabag
Loading composer repositories with package information
Installing dependencies
  - Installing mgargano/simplehtmldom (1.5)
    Loading from cache

  - Installing symfony/polyfill-util (v1.0.1)
    Downloading: 100%         

  - Installing paragonie/random_compat (1.1.5)
    Downloading: 100%         

  - Installing symfony/polyfill-php70 (v1.0.1)
    Downloading: 100%         

  - Installing symfony/polyfill-php56 (v1.0.1)
    Downloading: 100%         

  - Installing symfony/polyfill-mbstring (v1.0.1)
    Downloading: 100%         

  - Installing symfony/symfony (v3.0.1)
    Downloading: 100%         
PHP Fatal error:  Allowed memory size of 1073741824 bytes exhausted (tried to allocate 6710651 bytes) in phar:///usr/local/bin/composer.phar/src/Composer/Util/RemoteFilesystem.php on line 277

Fatal error: Allowed memory size of 1073741824 bytes exhausted (tried to allocate 6710651 bytes) in phar:///usr/local/bin/composer.phar/src/Composer/Util/RemoteFilesystem.php on line 277
```

**Comment by bdunogier** [comment]
Well, at least it has managed to resolve the dependencies, it is an improvement :)

**Comment by j0k3r** [comment]
@darkgeek @Asara Could you guys try with this command: `composer create-project wallabag/wallabag wallabag dev-v2-themes-folder` ?

This should fix your asset problem. Not the memory limit one :confused:

**Comment by darkgeek** [comment]
@j0k3r Well, I got the memory limit error again on my 1G computer :hushed::
```bash
PHP Fatal error:  Allowed memory size of 1073741824 bytes exhausted (tried to allocate 72 bytes) in phar:///usr/local/bin/composer.phar/src/Composer/DependencyResolver/RuleWatchGraph.php on line 104

Fatal error: Allowed memory size of 1073741824 bytes exhausted (tried to allocate 72 bytes) in phar:///usr/local/bin/composer.phar/src/Composer/DependencyResolver/RuleWatchGraph.php on line 104
```
I will try it on another 4G memory machine later :worried:.

**Comment by j0k3r** [comment]
@darkgeek don't worry this memory limit error will be a problem **only** for dev test. Real release won't have this problem (like you don't have this problem when using `2.0.0-alpha.1`).

**Comment by darkgeek** [comment]
@j0k3r Hi, I tried to install it on a 4G FreeBSD 10.2 computer with 6G memory, however, it complained another error:
```bash
% composer create-project wallabag/wallabag wallabag dev-v2-themes-folder
Installing wallabag/wallabag (dev-v2-themes-folder 30d81a47c689e1d7d963fcd3fd42af9958805e31)
  - Installing wallabag/wallabag (dev-v2-themes-folder v2-themes-folder)
    Cloning v2-themes-folder
    Failed to download wallabag/wallabag from source: The process "git clone --no-checkout 'git://github.com/wallabag/wallabag.git' 'wallabag/' && cd 'wallabag/' && git remote add composer 'git://github.com/wallabag/wallabag.git' && git fetch composer" exceeded the timeout of 300 seconds.
    Now trying to download from dist
  - Installing wallabag/wallabag (dev-v2-themes-folder v2-themes-folder)
    Loading from cache

Created project in wallabag

Fatal error: Call to undefined function JsonSchema\Constraints\filter_var() in phar:///usr/local/bin/composer.phar/vendor/justinrainbow/json-schema/src/JsonSchema/Constraints/FormatConstraint.php on line 83
```
Maybe I missed any dependencies?

**Comment by j0k3r** [comment]
Wow, which version of PHP are you using? Can you list loaded extension?

**Comment by darkgeek** [comment]
@j0k3r These are installed php and its extensions:
```bash
% pkg info | grep php
php-composer-1.0.0.a11_1       Dependency Manager for PHP
php56-5.6.17                   PHP Scripting Language
php56-ctype-5.6.17             The ctype shared extension for php
php56-gettext-5.6.17           The gettext shared extension for php
php56-hash-5.6.17              The hash shared extension for php
php56-json-5.6.17              The json shared extension for php
php56-mysql-5.6.17             The mysql shared extension for php
php56-openssl-5.6.17           The openssl shared extension for php
php56-pdo-5.6.17               The pdo shared extension for php
php56-pdo_mysql-5.6.17         The pdo_mysql shared extension for php
php56-pdo_sqlite-5.6.17        The pdo_sqlite shared extension for php
php56-phar-5.6.17              The phar shared extension for php
php56-sqlite3-5.6.17           The sqlite3 shared extension for php
php56-tidy-5.6.17              The tidy shared extension for php
php56-tokenizer-5.6.17         The tokenizer shared extension for php
php56-xml-5.6.17               The xml shared extension for php
```
And PHP modules are:
```bash
% php -m
[PHP Modules]
Core
ctype
date
ereg
gettext
hash
intl
json
libxml
mhash
mysql
mysqlnd
openssl
pcre
PDO
pdo_mysql
pdo_sqlite
Phar
Reflection
SPL
sqlite3
standard
tidy
tokenizer
xml
```

**Comment by j0k3r** [comment]
Found this https://gist.github.com/maciakl/5320119, could you try it ?

**Comment by Asara** [comment]
@j0k3r 
Having pretty much the same set of issues as @darkgeek, maybe it has to do with how fBSD is handling something on the backend.


    root@wallabag:/usr/local/www/apache24 # composer create-project wallabag/wallabag wallabag 2.x-dev
    Installing wallabag/wallabag (2.x-dev 124f8091cbb1e5cccd7972fa368f8e7c434e0e5b)
    - Installing wallabag/wallabag (2.x-dev 124f809)
    Cloning 124f8091cbb1e5cccd7972fa368f8e7c434e0e5b
    Created project in wallabag
    Loading composer repositories with package information
    Installing dependencies (including require-dev)
    PHP Fatal error:  Allowed memory size of 1073741824 bytes exhausted (tried to allocate 72 bytes) in phar:///usr/local/bin/composer.phar/src/Composer/DependencyResolver/Decisions.php on line 40

    Fatal error: Allowed memory size of 1073741824 bytes exhausted (tried to allocate 72 bytes) in phar:///usr/local/bin/composer.phar/src/Composer/DependencyResolver/Decisions.php on line 40

But my server has 16GB of ram...

When doing the dev-v2-themes folder I get the same error as @darkgeek.

**Comment by darkgeek** [comment]
@j0k3r Hi, I followed the gist provided and reinstall again, but I still have the same error as @Asara .

**Comment by j0k3r** [comment]
@darkgeek you mean, the memory limit?
If yes, could you run these and give me the output?

- `which composer`
- `php -r "phpinfo();" | grep memory`
- `php -r "phpinfo();" | grep .ini`

**Comment by darkgeek** [comment]
@j0k3r Yes, and the outputs are:
```bash
% which composer
/usr/local/bin/composer

% php -r "phpinfo();" | grep memory
memory_limit => 128M => 128M
Collecting memory statistics => No

% php -r "phpinfo();" | grep .ini
Configuration File (php.ini) Path => /usr/local/etc
Scan this dir for additional .ini files => /usr/local/etc/php
Additional .ini files parsed => /usr/local/etc/php/extensions.ini
user_ini.cache_ttl => 300 => 300
user_ini.filename => .user.ini => .user.ini
com_init_db => 0
Classes => AppendIterator, ArrayIterator, ArrayObject, BadFunctionCallException, BadMethodCallException, CachingIterator, CallbackFilterIterator, DirectoryIterator, DomainException, EmptyIterator, FilesystemIterator, FilterIterator, GlobIterator, InfiniteIterator, InvalidArgumentException, IteratorIterator, LengthException, LimitIterator, LogicException, MultipleIterator, NoRewindIterator, OutOfBoundsException, OutOfRangeException, OverflowException, ParentIterator, RangeException, RecursiveArrayIterator, RecursiveCachingIterator, RecursiveCallbackFilterIterator, RecursiveDirectoryIterator, RecursiveFilterIterator, RecursiveIteratorIterator, RecursiveRegexIterator, RecursiveTreeIterator, RegexIterator, RuntimeException, SplDoublyLinkedList, SplFileInfo, SplFileObject, SplFixedArray, SplHeap, SplMinHeap, SplMaxHeap, SplObjectStorage, SplPriorityQueue, SplQueue, SplStack, SplTempFileObject, UnderflowException, UnexpectedValueException
```

**Comment by j0k3r** [comment]
All right, could you try:

```
php -d memory_limit=-1 /usr/local/bin/composer create-project wallabag/wallabag wallabag dev-v2-themes-folder
```

This is an ugly fix to allow unlimited memory to PHP, it won't kill the machine :wink:

**Comment by darkgeek** [comment]
@j0k3r Thanks, it works!!! Love `wallabag` so much. :kissing_heart:

**Comment by darkgeek** [comment]
@j0k3r Well, I found one strange problem. I installed it using the methods above successfully, however, when I run command `php bin/console server:run 192.168.1.11:8000`, and after login from Firefox, the welcome page showed, but the `wallabag` folder on the server disappeared :scream:. Not sure how to deal with it.

The following is the output of installation:
```bash
% php -d memory_limit=-1 /usr/local/bin/composer create-project wallabag/wallabag wallabag dev-v2-themes-folder
Installing wallabag/wallabag (dev-v2-themes-folder 30d81a47c689e1d7d963fcd3fd42af9958805e31)
  - Installing wallabag/wallabag (dev-v2-themes-folder v2-themes-folder)
    Cloning v2-themes-folder
    Failed to download wallabag/wallabag from source: The process "git clone --no-checkout 'git://github.com/wallabag/wallabag.git' 'wallabag/' && cd 'wallabag/' && git remote add composer 'git://github.com/wallabag/wallabag.git' && git fetch composer" exceeded the timeout of 300 seconds.
    Now trying to download from dist
  - Installing wallabag/wallabag (dev-v2-themes-folder v2-themes-folder)
    Loading from cache

Created project in wallabag
Loading composer repositories with package information
Installing dependencies (including require-dev)
  - Installing mgargano/simplehtmldom (1.5)
    Loading from cache

  - Installing symfony/polyfill-util (v1.0.1)
    Loading from cache

  - Installing paragonie/random_compat (1.1.5)
    Loading from cache

  - Installing symfony/polyfill-php70 (v1.0.1)
    Loading from cache

  - Installing symfony/polyfill-php56 (v1.0.1)
    Loading from cache

  - Installing symfony/polyfill-mbstring (v1.0.1)
    Loading from cache

  - Installing symfony/symfony (v3.0.1)
    Loading from cache

  - Installing symfony/polyfill-intl-icu (v1.0.1)
    Loading from cache

  - Installing psr/log (1.0.0)
    Loading from cache

  - Installing twig/twig (v1.23.3)
    Loading from cache

  - Installing doctrine/lexer (v1.0.1)
    Loading from cache

  - Installing doctrine/annotations (v1.2.7)
    Loading from cache

  - Installing doctrine/collections (v1.3.0)
    Loading from cache

  - Installing doctrine/cache (v1.6.0)
    Loading from cache

  - Installing doctrine/inflector (v1.1.0)
    Loading from cache

  - Installing doctrine/common (v2.6.1)
    Loading from cache

  - Installing friendsofsymfony/user-bundle (dev-master 6ccff96)
    Cloning 6ccff96434c0ac7fee077d1dce90966341dfd278

  - Installing sonata-project/google-authenticator (1.0.2)
    Loading from cache

  - Installing scheb/two-factor-bundle (v2.0.0)
    Loading from cache

  - Installing behat/transliterator (v1.1.0)
    Loading from cache

  - Installing gedmo/doctrine-extensions (v2.4.10)
    Loading from cache

  - Installing stof/doctrine-extensions-bundle (dev-master 6672ca3)
    Cloning 6672ca37fe19a8e2a8ae52a93a7163e1b00ef7d5

  - Installing doctrine/doctrine-cache-bundle (1.2.2)
    Loading from cache

  - Installing twig/extensions (v1.3.0)
    Loading from cache

  - Installing kriswallsmith/assetic (v1.3.2)
    Loading from cache

  - Installing symfony/assetic-bundle (v2.7.1)
    Loading from cache

  - Installing swiftmailer/swiftmailer (v5.4.1)
    Loading from cache

  - Installing symfony/swiftmailer-bundle (v2.3.11)
    Loading from cache

  - Installing monolog/monolog (1.17.2)
    Loading from cache

  - Installing symfony/monolog-bundle (v2.8.2)
    Loading from cache

  - Installing sensiolabs/security-checker (v3.0.2)
    Loading from cache

  - Installing sensio/distribution-bundle (v5.0.3)
    Loading from cache

  - Installing sensio/framework-extra-bundle (v3.0.12)
    Loading from cache

  - Installing incenteev/composer-parameter-handler (v2.1.2)
    Loading from cache

  - Installing nelmio/cors-bundle (1.4.1)
    Loading from cache

  - Installing willdurand/jsonp-callback-validator (v1.1.0)
    Loading from cache

  - Installing willdurand/negotiation (1.5.0)
    Loading from cache

  - Installing friendsofsymfony/rest-bundle (1.7.7)
    Loading from cache

  - Installing michelf/php-markdown (1.6.0)
    Loading from cache

  - Installing nelmio/api-doc-bundle (2.11.2)
    Loading from cache

  - Installing ezyang/htmlpurifier (v4.7.0)
    Loading from cache

  - Installing doctrine/instantiator (1.0.5)
    Loading from cache

  - Installing phpoption/phpoption (1.5.0)
    Loading from cache

  - Installing phpcollection/phpcollection (0.4.0)
    Loading from cache

  - Installing jms/metadata (1.5.1)
    Loading from cache

  - Installing jms/parser-lib (1.0.0)
    Loading from cache

  - Installing jms/serializer (1.1.0)
    Loading from cache

  - Installing willdurand/hateoas (v2.9.0)
    Loading from cache

  - Installing jms/serializer-bundle (1.1.0)
    Loading from cache

  - Installing willdurand/hateoas-bundle (1.1.0)
    Loading from cache

  - Installing liip/theme-bundle (1.4.0)
    Loading from cache

  - Installing pagerfanta/pagerfanta (v1.0.3)
    Loading from cache

  - Installing doctrine/dbal (v2.5.4)
    Loading from cache

  - Installing doctrine/orm (v2.5.4)
    Loading from cache

  - Installing lexik/form-filter-bundle (v5.0.0)
    Loading from cache

  - Installing tecnickcom/tcpdf (6.2.12)
    Loading from cache

  - Installing smalot/pdfparser (v0.9.25)
    Loading from cache

  - Installing neitanod/forceutf8 (v2.0)
    Loading from cache

  - Installing htmlawed/htmlawed (1.1.19)
    Loading from cache

  - Installing simplepie/simplepie (1.3.1)
    Loading from cache

  - Installing j0k3r/graby-site-config (1.0.12)
    Loading from cache

  - Installing j0k3r/php-readability (v1.0.9)
    Loading from cache

  - Installing j0k3r/safecurl (2.0.2)
    Loading from cache

  - Installing react/promise (v2.2.1)
    Loading from cache

  - Installing guzzlehttp/streams (3.0.0)
    Loading from cache

  - Installing guzzlehttp/ringphp (1.1.0)
    Loading from cache

  - Installing guzzlehttp/guzzle (5.3.0)
    Loading from cache

  - Installing j0k3r/graby (1.2.0)
    Loading from cache

  - Installing friendsofsymfony/oauth2-php (1.2.0)
    Loading from cache

  - Installing friendsofsymfony/oauth-server-bundle (1.5.0)
    Loading from cache

  - Installing grandt/relativepath (1.0.2)
    Loading from cache

  - Installing grandt/binstring (1.0.0)
    Loading from cache

  - Installing grandt/phpresizegif (1.0.3)
    Loading from cache

  - Installing grandt/phpzipmerge (1.0.4)
    Loading from cache

  - Installing phpzip/phpzip (2.0.8)
    Loading from cache

  - Installing grandt/phpepub (4.0.3)
    Loading from cache

  - Installing wallabag/php-mobi (1.0.1)
    Loading from cache

  - Installing hoa/exception (1.16.01.11)
    Loading from cache

  - Installing hoa/consistency (1.16.01.14)
    Loading from cache

  - Installing hoa/event (1.16.01.11)
    Loading from cache

  - Installing hoa/protocol (1.16.01.11)
    Loading from cache

  - Installing hoa/stream (1.16.01.14)
    Loading from cache

  - Installing hoa/iterator (2.16.01.11)
    Loading from cache

  - Installing hoa/file (1.16.01.15)
    Loading from cache

  - Installing hoa/visitor (2.16.01.11)
    Loading from cache

  - Installing hoa/zformat (1.16.01.14)
    Loading from cache

  - Installing hoa/ustring (4.16.01.11)
    Loading from cache

  - Installing hoa/compiler (3.16.01.14)
    Loading from cache

  - Installing hoa/math (1.16.01.15)
    Loading from cache

  - Installing hoa/regex (1.16.01.15)
    Loading from cache

  - Installing hoa/ruler (2.16.01.15)
    Loading from cache

  - Installing kphoen/rulerz (0.18.0)
    Loading from cache

  - Installing kphoen/rulerz-bundle (0.12.0)
    Loading from cache

  - Installing doctrine/migrations (v1.2.2)
    Loading from cache

  - Installing jdorn/sql-formatter (v1.2.17)
    Loading from cache

  - Installing doctrine/doctrine-bundle (1.6.1)
    Loading from cache

  - Installing doctrine/doctrine-migrations-bundle (1.1.1)
    Loading from cache

  - Installing doctrine/data-fixtures (v1.1.1)
    Loading from cache

  - Installing doctrine/doctrine-fixtures-bundle (2.3.0)
    Loading from cache

  - Installing sensio/generator-bundle (v3.0.5)
    Loading from cache

  - Installing sebastian/version (1.0.6)
    Loading from cache

  - Installing sebastian/global-state (1.1.1)
    Loading from cache

  - Installing sebastian/recursion-context (1.0.2)
    Loading from cache

  - Installing sebastian/exporter (1.2.1)
    Loading from cache

  - Installing sebastian/environment (1.3.3)
    Loading from cache

  - Installing sebastian/diff (1.4.1)
    Loading from cache

  - Installing sebastian/comparator (1.2.0)
    Loading from cache

  - Installing phpdocumentor/reflection-docblock (2.0.4)
    Loading from cache

  - Installing phpspec/prophecy (v1.5.0)
    Loading from cache

  - Installing phpunit/php-text-template (1.2.1)
    Loading from cache

  - Installing phpunit/phpunit-mock-objects (2.3.8)
    Loading from cache

  - Installing phpunit/php-timer (1.0.7)
    Loading from cache

  - Installing phpunit/php-token-stream (1.4.8)
    Loading from cache

  - Installing phpunit/php-file-iterator (1.4.1)
    Loading from cache

  - Installing phpunit/php-code-coverage (2.2.4)
    Loading from cache

  - Installing phpunit/phpunit (4.8.21)
    Loading from cache

  - Installing symfony/phpunit-bridge (v2.8.2)
    Loading from cache

paragonie/random_compat suggests installing ext-libsodium (Provides a modern crypto API that can be used to generate random bytes.)
friendsofsymfony/user-bundle suggests installing willdurand/propel-typehintable-behavior (Needed when using the propel implementation)
gedmo/doctrine-extensions suggests installing doctrine/mongodb-odm (to use the extensions with the MongoDB ODM)
stof/doctrine-extensions-bundle suggests installing doctrine/mongodb-odm-bundle (to use the MongoDB ODM extensions)
doctrine/doctrine-cache-bundle suggests installing symfony/security-acl (For using this bundle to cache ACLs)
kriswallsmith/assetic suggests installing leafo/lessphp (Assetic provides the integration with the lessphp LESS compiler)
kriswallsmith/assetic suggests installing leafo/scssphp (Assetic provides the integration with the scssphp SCSS compiler)
kriswallsmith/assetic suggests installing ptachoire/cssembed (Assetic provides the integration with phpcssembed to embed data uris)
kriswallsmith/assetic suggests installing leafo/scssphp-compass (Assetic provides the integration with the SCSS compass plugin)
kriswallsmith/assetic suggests installing patchwork/jsqueeze (Assetic provides the integration with the JSqueeze JavaScript compressor)
symfony/assetic-bundle suggests installing kriswallsmith/spork (to be able to dump assets in parallel)
monolog/monolog suggests installing graylog2/gelf-php (Allow sending log messages to a GrayLog2 server)
monolog/monolog suggests installing raven/raven (Allow sending log messages to a Sentry server)
monolog/monolog suggests installing doctrine/couchdb (Allow sending log messages to a CouchDB server)
monolog/monolog suggests installing ruflin/elastica (Allow sending log messages to an Elastic Search server)
monolog/monolog suggests installing videlalvaro/php-amqplib (Allow sending log messages to an AMQP server using php-amqplib)
monolog/monolog suggests installing ext-amqp (Allow sending log messages to an AMQP server (1.0+ required))
monolog/monolog suggests installing ext-mongo (Allow sending log messages to a MongoDB server)
monolog/monolog suggests installing aws/aws-sdk-php (Allow sending log messages to AWS services like DynamoDB)
monolog/monolog suggests installing rollbar/rollbar (Allow sending log messages to Rollbar)
monolog/monolog suggests installing php-console/php-console (Allow sending log messages to Google Chrome)
sensio/framework-extra-bundle suggests installing symfony/psr-http-message-bridge (To use the PSR-7 converters)
nelmio/api-doc-bundle suggests installing dunglas/api-bundle (For making use of resources definitions of DunglasApiBundle.)
jms/serializer-bundle suggests installing jms/di-extra-bundle (Required to get lazy loading (de)serialization visitors, ~1.3)
pagerfanta/pagerfanta suggests installing mandango/mandango (To use the MandangoAdapter.)
pagerfanta/pagerfanta suggests installing doctrine/mongodb-odm (To use the DoctrineODMMongoDBAdapter.)
pagerfanta/pagerfanta suggests installing doctrine/phpcr-odm (To use the DoctrineODMPhpcrAdapter. >= 1.1.0)
pagerfanta/pagerfanta suggests installing propel/propel1 (To use the PropelAdapter)
pagerfanta/pagerfanta suggests installing solarium/solarium (To use the SolariumAdapter.)
friendsofsymfony/oauth-server-bundle suggests installing doctrine/mongodb-odm-bundle (*)
friendsofsymfony/oauth-server-bundle suggests installing propel/propel-bundle (If you want to use Propel with Symfony2, then you will have to install the PropelBundle)
friendsofsymfony/oauth-server-bundle suggests installing willdurand/propel-typehintable-behavior (The Typehintable behavior is useful to add type hints on generated methods, to be compliant with interfaces)
kphoen/rulerz suggests installing kphoen/rulerz-spec-builder (If you want a specification builder)
kphoen/rulerz suggests installing elasticsearch/elasticsearch (To execute rules as Elasticsearch queries)
kphoen/rulerz suggests installing pomm-project/model-manager (To execute rules as Pomm queries)
kphoen/rulerz suggests installing solarium/solarium (To execute rules as Solr queries)
doctrine/data-fixtures suggests installing doctrine/mongodb-odm (For loading MongoDB ODM fixtures)
doctrine/data-fixtures suggests installing doctrine/phpcr-odm (For loading PHPCR ODM fixtures)
sebastian/global-state suggests installing ext-uopz (*)
phpdocumentor/reflection-docblock suggests installing dflydev/markdown (~1.0)
phpdocumentor/reflection-docblock suggests installing erusev/parsedown (~1.0)
phpunit/phpunit-mock-objects suggests installing ext-soap (*)
phpunit/php-code-coverage suggests installing ext-xdebug (>=2.2.1)
phpunit/phpunit suggests installing phpunit/php-invoker (~1.1)
Writing lock file
Generating autoload files
> Incenteev\ParameterHandler\ScriptHandler::buildParameters
Creating the "app/config/parameters.yml" file
Some parameters are missing. Please provide them.
database_driver (pdo_sqlite): 
database_host (127.0.0.1): 
database_port (null): 
database_name (symfony): 
database_user (root): 
database_password (null): 102030
database_path ('%kernel.root_dir%/../data/db/wallabag.sqlite'): 
database_table_prefix (wallabag_): 
test_database_driver (pdo_sqlite): 
test_database_host (127.0.0.1): 
test_database_port (null): 
test_database_name (null): wallabag_test
test_database_user (null): root
test_database_password (null): 102030
test_database_path ('%kernel.root_dir%/../data/db/wallabag_test.sqlite'): 
mailer_transport (smtp): 
mailer_host (127.0.0.1): 
mailer_user (null): 
mailer_password (null): 
locale (en): 
secret (ThisTokenIsNotSoSecretChangeIt): 102030
app.version (2.0.0-alpha): 
twofactor_auth (true): 
twofactor_sender (no-reply@wallabag.org): 
warning_message ("You're trying wallabag v2, which is in alpha version. If you find a bug, please have a look to <a href=\"https://github.com/wallabag/wallabag/issues\">our issues list</a> and <a href=\"https://github.com/wallabag/wallabag/issues/new\">open a new if necessary</a>\n"): 
download_pictures (false): 
share_twitter (true): 
share_mail (true): 
share_shaarli (true): 
shaarli_url ('http://myshaarli.com'): 
share_diaspora (true): 
diaspora_url ('http://diasporapod.com'): 
flattr (true): 
carrot (true): 
show_printlink (true): 
export_epub (true): 
export_mobi (true): 
export_pdf (true): 
wallabag_url ('http://v2.wallabag.org'): 
wallabag_support_url ('https://www.wallabag.org/pages/support.html'): 
items_on_page (12): 
theme (material): 
language (en): 
from_email (no-reply@wallabag.org): 
rss_limit (50): 
pocket_consumer_key (xxxxxxxx): 
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::buildBootstrap
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::clearCache

 // Clearing the cache for the dev environment with debug true

                                                                                                                        
 [OK] Cache for the "dev" environment (debug=true) was successfully cleared.                                            
                                                                                                                        

> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::installAssets

 Trying to install assets as relative symbolic links.

 --- -------------------- ------------------ 
      Bundle               Method / Error    
 --- -------------------- ------------------ 
  ✔   FrameworkBundle      relative symlink  
  ✔   NelmioApiDocBundle   relative symlink  
  ✔   WallabagCoreBundle   relative symlink  
 --- -------------------- ------------------ 

                                                                                                                        
 [OK] All assets were successfully installed.                                                                           
                                                                                                                        

> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::installRequirementsFile
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::prepareDeploymentTarget

% php bin/console wallabag:install
Installing Wallabag...

Step 1 of 4. Checking system requirements.
+---------+--------+----------------+
| Checked | Status | Recommendation |
+---------+--------+----------------+
| PCRE    | OK!    |                |
| DOM     | OK!    |                |
+---------+--------+----------------+
Success! Your system can run Wallabag properly.

Step 2 of 4. Setting up database.
Creating database and schema, clearing the cache

Step 3 of 4. Administration setup.
Would you like to create a new user ? (y/N)y
Username (default: wallabag) :  
Password (default: wallabag) :
Email:

Step 4 of 4. Installing assets.

Wallabag has been successfully installed.
Just execute `php bin/console server:run` for using wallabag: http://localhost:8000

php bin/console server:run 192.168.1.11:8000

                                                                                                                        
 [OK] Server running on http://192.168.1.11:8000                                                                      
                                                                                                                        

 // Quit the server with CONTROL-C.
```

**Comment by j0k3r** [comment]
Glad it worked!
Well, for sure, there is no such thing that remove the folder after first usage :)

@Asara could you try the command posted [here](https://github.com/wallabag/wallabag/issues/1613#issuecomment-173972172) ?

**Comment by darkgeek** [comment]
@j0k3r Yep, that's my server's internal problem, not `wallabag`, sorry for that :sweat_smile:.

**Comment by Asara** [comment]
@j0k3r 
Hey the command works.

I have a unrelated question.  Is there a way of enabling users without using smtp to authenticate?

I am only going to be using this 2 or 3 people so enabling accounts manually isn't an issue for me.

**Comment by j0k3r** [comment]
@Asara Gerat!
What do you mean by "without using smtp to authenticate" ? User can log in using username. We don't send an email, only for forgotten password.

**Comment by Asara** [comment]
@j0k3r Hm... Doesn't make sense to me.

    An email has been sent to X@Y.com. It contains an activation link you must click to activate your account.

Happens when you create a new account.

    User account is disabled. 

When you try to sign in without authenticating.

If I enable the user manually in the database:

    MariaDB [(none)]> use wallabag
    Reading table information for completion of table and column names
    You can turn off this feature to get a quicker startup with -A

    Database changed
    MariaDB [wallabag]> update wallabag_user set enabled=1 where username='username';
    Query OK, 1 row affected (0.02 sec)
    Rows matched: 1  Changed: 1  Warnings: 0

Then:

    INFO - Matched route "fos_user_security_check".
    DEBUG - SELECT t0.username AS username_1, t0.username_canonical AS username_canonical_2, t0.email AS email_3, t0.email_canonical AS email_canonical_4, t0.enabled AS enabled_5, t0.salt AS salt_6, t0.password AS password_7, t0.last_login AS last_login_8, t0.locked AS locked_9, t0.expired AS expired_10, t0.expires_at AS expires_at_11, t0.confirmation_token AS confirmation_token_12, t0.password_requested_at AS password_requested_at_13, t0.roles AS roles_14, t0.credentials_expired AS credentials_expired_15, t0.credentials_expire_at AS credentials_expire_at_16, t0.id AS id_17, t0.name AS name_18, t0.created_at AS created_at_19, t0.updated_at AS updated_at_20, t0.authCode AS authCode_21, t0.twoFactorAuthentication AS twoFactorAuthentication_22, t0.trusted AS trusted_23, t24.id AS id_25, t24.theme AS theme_26, t24.items_per_page AS items_per_page_27, t24.language AS language_28, t24.rss_token AS rss_token_29, t24.rss_limit AS rss_limit_30, t24.user_id AS user_id_31 FROM `wallabag_user` t0 LEFT JOIN `wallabag_config` t24 ON t24.user_id = t0.id WHERE t0.username_canonical = ? LIMIT 1
    INFO - User has been authenticated successfully.
    CRITICAL - Fatal Error: Call to a member function getLanguage() on null
    CRITICAL - Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function getLanguage() on null" at /usr/local/www/apache24/wallabag/src/Wallabag/CoreBundle/EventListener/UserLocaleListener.php line 33
    DEBUG - Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". 

The DEBUG - Notfied event "kernel..." a bunch of times and then

    DEBUG - Notified event "kernel.controller" to listener "FOS\RestBundle\EventListener\ParamFetcherListener::onKernelController".
    DEBUG - Notified event "kernel.controller" to listener "Symfony\Bundle\FrameworkBundle\DataCollector\RouterDataCollector::onKernelController".
    DEBUG - Notified event "kernel.controller" to listener "Symfony\Component\HttpKernel\DataCollector\RequestDataCollector::onKernelController".
    DEBUG - Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ControllerListener::onKernelController".
    DEBUG - Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener::onKernelController".
    DEBUG - Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelController".
    DEBUG - Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\SecurityListener::onKernelController".
    DEBUG - Notified event "kernel.controller" to listener "FOS\RestBundle\EventListener\ViewResponseListener::onKernelController".

**Comment by j0k3r** [comment]
@Asara Oh yeah, I'm sorry. I've forgotten about that.
Could you open a separate issue about that?
We still a need a little admin page to manage user.

**Comment by Asara** [comment]
@j0k3r No doubt.  Thanks man!
